data/raw.dvc

```diff
@@ -1,5 +1,5 @@
 outs:
-- md5: df1762135f5161b135cc4e664242b1ea.dir
-  size: 437588778
-  nfiles: 364
+- md5: 0da22d1871320983c53fe8182e55f514.dir
+  size: 26380
+  nfiles: 1
   path: raw
```

Comment:
Oops - this should not have been pushed w/o pulling from data/raw first.

Reply:
Looks like it's fixed!